protoc-gen-twirp/generator.go

```diff
@@ -61,14 +64,43 @@ func newGenerator() *twirp {
 	t := &twirp{
 		pkgs:                make(map[string]string),
 		pkgNamesInUse:       make(map[string]bool),
+		importMap:           make(map[string]string),
 		fileToGoPackageName: make(map[*descriptor.FileDescriptorProto]string),
 		output:              bytes.NewBuffer(nil),
 	}
 
 	return t
 }
 
+// commandLineParameters breaks the comma-separated list of key=value pairs
+// in the parameter (a member of the request protobuf) into a key/value map.
+// It then sets file name mappings defined by those entries.
+func (t *twirp) commandLineParameters(parameter string) {
+	param := make(map[string]string)
+	for _, p := range strings.Split(parameter, ",") {
+		if i := strings.Index(p, "="); i < 0 {
+			param[p] = ""
+		} else {
+			param[p[0:i]] = p[i+1:]
+		}
+	}
+
+	t.importMap = make(map[string]string)
+	for k, v := range param {
+		switch k {
+		case "import_prefix":
+			t.importPrefix = v
+		default:
+			if len(k) > 0 && k[0] == 'M' {
```

Comment:
Ugh, this is a terrible API for users... but it _is_ what `github.com/golang/protobuf/protoc-gen-go` does. I suppose we should support it.

But can we also provide a less-insane format? I don't know exactly what, but here's an idea:
```
protoc --twirp_out="go_package@rpc/empty.proto=github.com/twitchtv/rpc/empty,go_package@something/else.proto=golang.org/x/net/notreal.proto:.
```